app/controllers/mixins/ems_common_angular.rb

```diff
@@ -431,6 +431,7 @@ def build_credentials(ems, mode)
          ems.supports_authentication?(:bearer) && !params[:default_password].blank?
         creds[:hawkular] = {:auth_key => params[:default_password], :userid => "_"}
         creds[:bearer] = {:auth_key => params[:default_password]}
+        creds.delete(:default)
```

Comment:
IMO the `build_credentials` method needs a *lot* of work, of which Containers is going to be just one part.

Looking up higher in the method reveals something of a Jenga game of stacking logic.

Let's stick with how @yaacov has this for now.  And, put this on the list for refactoring (cc @juliancheal, add this to your list :) )